pygluu/kubernetes/gui/views/install.py

```diff
@@ -0,0 +1,63 @@
+import threading
+import os
+from flask import Blueprint, render_template
+from flask_socketio import emit
+from pygtail import Pygtail
+from queue import Queue
+
+from pygluu.kubernetes.kustomize import Kustomize
+from pygluu.kubernetes.helm import Helm
+from pygluu.kubernetes.settings import SettingsHandler
+from ..extensions import socketio
+
+install_blueprint = Blueprint('install', __name__, template_folder="templates")
+settings = SettingsHandler()
+timeout = 120
+queue = Queue()
+
+
+@install_blueprint.route('/install')
+def install_kustomize():
+    t = threading.Thread(target=do_installation, args=(queue,))
+    t.daemon = True
+    t.start()
+    return render_template('install/kustomize.html')
+
+
+@socketio.on('install', namespace='/test')
+def installer_logs():
+    data = {}
+    tail_log = True
+    while tail_log:
+        if not queue.empty():
+            data = queue.get()
+
+        logs = Pygtail("./setup.log", read_from_end=True, every_n=1)
+        for log in logs.readlines():
+            emit('response', {'title': data[0], 'log': log, 'status': data[1]})
+
+        if data[1] == 'COMPLETED':
```

Comment:
Not sure im following the logic correctly. A key error can result if resources previously existed.